src/content/en/updates/2017/06/budget-api.md

```diff
@@ -37,7 +37,7 @@ visibility. The user agent will be responsible for determining budget
 assigned to a web app based on it's heuristics, for example budget allowance
 could be linked to user engagement. Each browser can decide it's own heuristic.
 
-**TL;DR **The budget API allows to you to reserve budget, use budget, get a list
+**TL;DR **The Budget API allows to you to reserve budget, use budget, get a list
```

Comment:
Yes, I capitalized it. Or was this for the benefit of people who don't know why I capitalized it? Thanks